src/Controller/GeneratorController.php

```diff
@@ -91,11 +92,18 @@ private function hydrateProject(array $formData): Project
         $appData    = $formData['applicationOptions'];
         $appOptions = new Application(applicationType: $appData['applicationType'], uploadSize: $appData['uploadSize']);
 
+        $workingDirData    = $formData['workingDirOptions'];
+        $workingDirOptions = new WorkingDir(
+            localWorkingDir: $workingDirData['localWorkingDir'],
+            dockerWorkingDir: $workingDirData['dockerWorkingDir']
```

Comment:
Did you check what happens when users add a trailing slash at the end of those two values in the form? We might possibly need to strip any trailing slashes (eg using `rtrim($value, '/')`) here to make sure we don't end up later on with invalid paths on the generated project.